gpt_index/data_structs/data_structs.py

```diff
@@ -397,6 +406,14 @@ def get_node_ids(self, keyword: str, depth: int = 1) -> List[str]:
             # TODO: Traverse (with depth > 1)
         return node_ids
 
+    def should_use_keywords(self) -> bool:
+        """Check if struct keywords should be used."""
+        return self._mode.__contains__(MODE_KEYWORDS)
```

Comment:
we shouldn't be using `__contains__` directly, we should be using `in` e.g. `MODE_KEYWORDS in self._mode`

Reply:
Ended up removing this entirely from the index_struct :)